packages/patternlab/styleguide/source/_patterns/05-pages/marketing-campaign~as-half-image.json

```
@@ -0,0 +1,171 @@
+{
+  "preContent": {
+    "keyMessage": {
+      "id": "abc1",
+      "positionTop": true,
+      "compHeading": {
+        "title": "Technology Careers at the Commonwealth of Massachusetts.",
+        "level": "1"
+      },
+      "halfImage": true,
+      "textOverlay": "c-gray-lightest",
+      "bgImage": {
+        "path": "@atoms/09-media/picture-element.twig",
+        "data": {
+          "picture": {
+            "defaultImage": "../../assets/images/placeholder/460x220.png",
+            "srcSets": [
+              {
+                "url": "../../assets/images/placeholder/1600x600.png",
+                "width": "1201"
+              },
+              {
+                "url": "../../assets/images/placeholder/900x700.png",
+                "width": "911"
+              },
+              {
+                "url": "../../assets/images/placeholder/500x300.png",
+                "width": "481"
+              }
+            ]
+          }
+        }
+      },
+      "mobileBgImage": "../../assets/images/placeholder/1600x600-lighthouse.jpg",
+      "subtitle": "From education, to healthcare and beyond, the work of more than 150 agencies across Massachusetts touches every aspect of our lives. We are a diverse team of mission-driven IT professionals supporting this vital work.",
```

```suggestion
      "subtitle": "From education, to healthcare and beyond, the work of more than 150 agencies across Massachusetts touches every aspect of our lives. We are a diverse team of mission-driven IT professionals supporting this vital work.",
```

can you remove the subtitle to show a more compact version of the header block using the half image version?

---

This was addressed by Luis.